about.html

```diff
@@ -0,0 +1,122 @@
+    rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <div>
+      <ul>
+        <li><i class="fa-brands fa-facebook-f"></i></li>
+        <li><i class="fa-brands fa-twitter"></i></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <div>
+      <img src="./assets/logo-talent-land.png" alt="">
+      <span id="openOnClick"><a href="javascript:void(0);" class="icon" onclick="myFunction()">&#9776</a></span>
+      <nav>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li>Program</li>
+          <li>Join</li>
+          <li>Sponsor</li>
+          <li>News</li>
+        </ul>
+      </nav>
+      <div class="dropdown">
+        <ul id="options" class="dropdown-content">
+          <li><a href="./index.html">Home</a></li>
+          <li>Program</li>
+          <li>Join</li>
+          <li>Sponsor</li>
+          <li>News</li>
+        </ul>
+      </div>
+      <div>
+        <p>CC Campaing</p>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="about_headline">
+      <div class="container">
+        <div>
+          <h1>Talent Land Mexico 2023</h1>
+        </div>
+        <div class="about_text">
+          <p>Talent Land 2023 will be dedicated to the development of talent, so that the participants can meet with
+            experts from different areas who will share their experience and knowledge.
+            This event is a place for creative minds, entrepreneurs and young talents to find new ideas, new people and
+            new technologies to improve their personal and professional lives.
+            Talent Land 2023 is underway and it will be unique.</p>
+        </div>
+        <div class="about_credit">
+          <p>Please contact us per email for any further questions about CC Talent Land Mexico.</p>
+          <p class="email">info@talent-network.org</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="about_talent container">
+      <h2>Jalisco Talent Land </h2>
+      <hr>
+      <p>Since the first Talent Land we managed to bring together creativity and technology, for the next 2023, the
+        machinery has been set in motion to create new emotions, promote the development of talent and promote
+        entrepreneurship among young people.
+        With the same entrepreneurial spirit that has allowed us to carry out unforgettable experiences on a global
+        level, in April 2023 we will discover new ways to change the world with talent.
+        “Talent Land 2023: the future is yours”
+        Get ready!</p>
+      <div class="about_logo">
+        <img src="./assets/logo-talent-land.png" alt="Talent Land Logo">
+      </div>
+    </section>
+
+    <section class="past-editions">
+      <div class="about-title">
+        <h2>See the past editions of Talent Land</h2>
+        <hr>
```

Comment:
- [ ] like the previous reviewer noted, you are missing the partners section in the mobile version of the about page. Kindly add. 
- [ ] the background of the mobile about page footer is white 
- [optional] please the logo and text on the footer is side by side and not one under the other. please adjust that for both pages
yours
![Screenshot (987)](https://user-images.githubusercontent.com/65298048/233717023-bfa9bcaf-b4fd-4ba7-9b5b-1cd0baf9410f.png)
original
![Screenshot (988)](https://user-images.githubusercontent.com/65298048/233717601-e300e730-ca09-43ba-a973-0958e485dba6.png)